client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -22,16 +20,27 @@ import PendingGappsTosNotice from './pending-gapps-tos-notice';
 import purchasesPaths from 'me/purchases/paths';
 import { type as domainTypes, transferStatus } from 'lib/domains/constants';
 import { isSubdomain, hasPendingGoogleAppsUsers } from 'lib/domains';
-import support from 'lib/url/support';
-import paths from 'my-sites/domains/paths';
+import {
```

Comment:
using named exports really isn't important in this context and I think i prefer the old name-spaced way of accessing these constants.

support.MAP_SUBDOMAIN over just a bare MAP_SUBDOMAIN.

🤷‍♂️

Reply:
we have that option to `import * as support` here but the reason I chose named exports is because for the most part they're all separate things. the reason I split them up is because I wanted to have it so that if a module only needs one then it won't have to package all of the exports into that bundle. I'm not sure if this actually happens but since it's kind of ubiquitous throughout the app I'm guessing that some bundles will only pull in a few of the exports.